pkg/v1/providers/tests/clustergen/param_models/cluster_optional.model

```diff
@@ -93,7 +93,9 @@ VSPHERE_CLONE_MODE: "fullClone","linkedClone"
 # http proxy (too many "NA" leads to overwhelmingly negative cases)
 TKG_HTTP_PROXY: "http://10.0.200.100", "http://[fc00:f853:ccd:e793::1]:3128"
 TKG_HTTPS_PROXY: "NA", "http://10.0.200.100", "http://[fc00:f853:ccd:e793::1]:3128"
-TKG_NO_PROXY: "NA", "10.0.200.1/24", "10.184.90.80"
+TKG_NO_PROXY: '"NA", "10.0.200.1/24", "10.184.90.80"
```

Comment:
this doesn't look right, possible values should belong on one line. Comma is a special character that needs to be escaped, so if the third case is to introduce a string with space, it should be:
```
TKG_NO_PROXY: '"NA", "10.0.200.1/24", "10.184.90.80<comma> 10.0.0.1/24"
```

Reply:
I want to add a new line. Sorry didnt know the syntax